modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -30,6 +30,7 @@ The following Technology Preview features are enabled by this feature set:
 ** Pod disruption budget (PDB) unhealthy pod eviction policy. Enables support for specifying how unhealthy pods are considered for eviction when using PDBs. (`PDBUnhealthyPodEvictionPolicy`)
 ** Dynamic Resource Allocation API. Enables a new API for requesting and sharing resources between pods and containers. This is an internal feature that most users do not need to interact with. (`DynamicResourceAllocation`)
 ** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
+** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
```

Comment:
Do the edits in this file need to go in the `main` branch as well? @cbippley

Reply:
Yes, I'll remove those changes and put them in a different PR. Sorry about that.

Reply:
no worries, feel free to ping me when the PR is up and i'll be happy to merge 🙂

Reply:
Fixed this PR so it is good to go, I'll ping when the next is ready. Thanks!